app/controllers/suggestions.rb

```diff
@@ -0,0 +1,82 @@
+get "/suggestions" do
+  redirect "/"
+end
+
+get "/suggestions/new" do
+  current_user
```

Comment:
This call to `current_user` is not needed. Only use this helper method if you intend on utilizing its return value. (This note applies to all lines that call this method without utilizing its return value.)